packages/evm-adapter/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "exclude": ["dist", "test"],
+  "include": ["."],
+  "compilerOptions": {
+    "target": "ES2020" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
You can remove the comments, I think it's cleaner that way

Reply:
OK